### PR TITLE
windows and arm64 platforms

### PR DESCRIPTION
**Pull request by loriab**
As it says on the tin, this adds two platforms. Associated upstream PR is linked below.

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] ~Reset the build number to `0` (if the version changed)~
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by loriab** [comment]
@conda-forge-admin, please rerender

**Comment by loriab** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml
Comment (jan-janssen) [inline recipe/meta.yaml]: The built seems to work now, can we get this merged up-stream?

Comment (loriab) [reply recipe/meta.yaml]: I'm cleaning up my changes in psiecp2023 to propose them to upstream and patch them to here. Mostly figuring out how much I must/can/should overhaul their cmake. For instance, I bet libecpint w/pugixml is only consumable through `find_package()` when the xml library is static (as it is for c-f). I've mostly been sticking to `cmake_minimum_required(VERSION 3.16)` -- do you have any cmake constraints?

Comment (jan-janssen) [reply recipe/meta.yaml]: Great. I am happy to merge this once the changes are merged up-stream. Just ping me again.

**Comment by loriab** [comment]
@conda-forge-admin, please rerender

**Comment by loriab** [comment]
Ok @jan-janssen, I think this is ready to go. The minimal changes have been collected into a patch here and PR'd to upstream. Your choice whether to consider this before upstream deliberates.

**Review by jan-janssen — APPROVED** [review]
Looks good to me

**Pull request merged**
